### PR TITLE
Dedicated Servers - Windows rescue + Windows admin password - Updates of screenshots

### PR DESCRIPTION
**Pull request by DaMikeTheOne**
Updates some screenshots for more accurate ones.

### CODE REVIEW

File: pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/images/ntpwedit_2.png
Comment (DaMikeTheOne) [inline pages/bare_metal_cloud/dedicated_servers/rcw-changing-admin-password-on-windows/images/ntpwedit_2.png:1]: Changes with new screenshot:
Added the asterisk characters to illustrate the need to enter a password according to explanations.

File: pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows/images/manager-rescue-windows-menu.png
Comment (DaMikeTheOne) [inline pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows/images/manager-rescue-windows-menu.png:1]: Changes with new screenshot:
Selection boxes are swapped. No SSH key authent available for Windows rescue. Rescue name is now accurate.

File: pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows/images/rescue-mode-001.png
Comment (DaMikeTheOne) [inline pages/bare_metal_cloud/dedicated_servers/rescue-customer-windows/images/rescue-mode-001.png:1]: Changes with new screenshot:
OS name displayed is no more a short name (and doesn't look forged: there is no 32bits version of win2022)
Edit button highlighting.

**Review by Y0Coss — APPROVED** [review]

**Comment by Y0Coss** [comment]
Thank you for your contribution @DaMikeTheOne

**Pull request merged**